### PR TITLE
Added login router

### PR DESCRIPTION
**Pull request by jayaregalinada**

### CODE REVIEW

File: routes/login-router.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    const { password } = request.body;
+
+    if (password !== find.password) {
+        // If exist, check yung password match from request
+        // Decode password from user object, check if it match
+        // If not match, response Invalid credential or 400 bad request
+        return response.status(400).json({error: 'Incorrect credentials'});
+    }
+
+    createSession(find);
```
Comment (jayaregalinada) [inline routes/login-router.js:27]: What the hell is this.

File: routes/login-router.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const router = Router();
+
+// POST /login
+router.post("/login", (request, response) => {
+    // User find username
+    const { username } = request.body;
+    const find = User.find({username});
+
+    if (find === null) {
```
Comment (haidee-tan) [inline routes/login-router.js:12]: sample comment

File: routes/login-router.js (hunk 1 above)
Comment (haidee-tan) [inline routes/login-router.js:27]: sample 2

File: routes/login-router.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const { Router } = require('express');
+const User = require('../models/User');
+
+const router = Router();
+
+// POST /login
+router.post("/login", (request, response) => {
```
Comment (haidee-tan) [inline routes/login-router.js:7]: sample

**Review by haidee-tan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**